src/Microsoft.Owin.Security.Cookies.Shareable/AspNet5TicketSerializer.cs

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Security.Claims;
+using Microsoft.Framework.Internal;
+using Microsoft.Owin.Security.DataHandler.Serializer;
+
+namespace Microsoft.Owin.Security.Cookies.Shareable
+{
+    public class AspNet5TicketSerializer : IDataSerializer<AuthenticationTicket>
```

Comment:
The name is a bit misleading.  The TicketSerializer should be reusable anywhere, the only thing that really interacts with cookies is via the sugar in UseCookieAuthentication.  We can certainly revisit how things are laid out/factored.